docs/runtimes/functions.md

```diff
@@ -375,6 +375,7 @@ common function methods:
 ```{toctree}
 :maxdepth: 2
 
+mlrun_code_annotations
 code-archive
```

Comment:
should add links from the relevant text section above (e.g. near the doc for `code_to_function()` )